be/README.md

```diff
@@ -1,54 +1,66 @@
-# TPT-Loane Backend
+# Back end
 
+## Table of contents
 
-## Installation
+- [Important variables](#important-variables)
 
-```bash
-$ npm install
-```
-## Database migrations 
+- [Running the development back end](#running-the-development-back-end)
 
-Generate migrations:  ```npm run gen-migration <ModelName>```
+- [Database migrations](#database-migrations)
 
-Run migrations: ```npm run migrate```
+- [Tests](#tests)
 
+- [Stopping the development back end](#stopping-the-development-back-end)
 
-## Running the app
+- [Troubleshooting](#troubleshooting)
 
-```bash
-# development
-$ npm run start
+## Important variables
+
+(no important variables specified)
```

Comment:
There are some very important env variables in both fe and be They can be found in the `.env.example` files. Please link to them as well.

Reply:
Fixed in new commit: 54e1249614707588a8a1562d9ac515df42d9ec53.